sparse_strips/vello_api/Cargo.toml

```diff
@@ -15,7 +15,10 @@ publish = false
 peniko = { workspace = true }
 
 [features]
+default = ["std"]
```

Comment:
Probably worth adding some basic docs here (i.e. a comment above the libm/std features)

I'd probably also lean towards grouping `libm` and `std`.